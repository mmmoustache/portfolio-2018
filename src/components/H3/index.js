import styled from "styled-components";
import { convertRemToPixels as rem } from '../../utils/utils';

export default styled.h3`
  font-size: ${rem('24px')};
  text-transform: uppercase;

  ${ 
    props => props.noMargin ? `margin: 0;` : `margin: 0 0 20px;`
  }
`;
