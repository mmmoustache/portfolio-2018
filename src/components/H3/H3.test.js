import React from 'react';
import renderer from 'react-test-renderer';
import H3 from "../H3";

describe('Component: H3', () => {

  it('Should render successfully', () => {
    const component = renderer.create(
      <H3>Test</H3>
    );

    expect(component.toJSON()).toMatchSnapshot();
  });

});
