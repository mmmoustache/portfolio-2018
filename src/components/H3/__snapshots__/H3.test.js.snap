// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: H3 Should render successfully 1`] = `
<h3
  className="sc-bdVaJa ccvsac"
>
  Test
</h3>
`;
